app/tests/spec/views/index.js

```diff
@@ -58,6 +57,9 @@ define((require, exports, module) => {
       view = null;
     });
 
+    it('getViewName returns `enter-email`', () => {
+      assert.equal(view.getViewName(), 'enter-email');
```

Comment:
Fwiw, I'm generally not a fan of unit-testing properties indirectly, via methods on other objects. It means you're coupling unit tests with objects other than the one they're testing. Any reason why this assertion can't be against `view.viewName` directly?